features/console.feature

```diff
@@ -1,6 +1,21 @@
 Feature: The console command
   Most aspects of connection to a server are tested in shell feature.
 
+  Scenario: Opening a remote Rails console with preconfigured irb flags from the global settings
+    Given a file named "tmp/global_settings.yml" with "irb_flags: --foo --baz"
+      And a file named "Capfile" with "Capfile exists"
+      And a file named "config/deploy.rb" with "deploy file exists"
+      And a file named "config/deploy/staging.rb" with:
+      """
+      set :deploy_to, '/var/www/example.com'
+      set :user, 'user'
+      server 'www.example.com'
+      """
+    When I run `geordi console staging`
+    Then the output should contain "# Opening a Rails console on staging"
+      And the output should contain "Util.run! ssh, user@www.example.com, -t, cd /var/www/example.com/current && bash --login -c 'bundle exec rails console -e  -- --foo --baz"
```

Comment:
There is the environment missing after `-e` (shouldn't it be "staging"?), and/or a double space present. Please show how `--nomultiline` is integrated.

Reply:
You are right. The environment is pulled out of the capistrano config. In the test setup the `rails_env` is missing in `config/deploy/staing.rb`. I have adjusted the test setup accordingly.